### PR TITLE
Math Magicians: App Components

### PR DESCRIPTION
**Pull request by ogagaoghene**
# Math Magicians: Component 

## Desctiption
- Creating components for the Math magician calculator app

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,83 @@
...
+            </div>
+            <div className="btn2 btnr2">
+              <input type="button" value="2" />
+            </div>
+            <div className="btn3 btnr2">
+              <input type="button" value="3" />
+            </div>
+            <div className="btnA btnr1">
+              <input type="button" value="+" />
+            </div>
```
Comment (youmari) [inline src/index.js:66]: - [ ] as stated in the requirements you should create a folder called `Components` and also a `Calculator` Component and move your calculator design into that component and it should be class based 
Screen Shot 2022-08-03 at 11 01 18

**Review by youmari — CHANGES_REQUESTED** [review]
Hi  @ogagaoghene , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✅ PRO PR
✅ Git flow
✅ Linters passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,83 @@
...
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-);
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```
[the remark this answers is not in the document]
Comment (ogagaoghene) [reply src/index.js]: Resolved as advised

**Comment by ogagaoghene** [comment]
I have created the folder and moved the file into it as requested

**Review by prolajumokeoni — APPROVED** [review]
Hi @ogagaoghene, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ogagaoghene** [comment]
Thanks

**Pull request merged**